ocs_ci/ocs/cluster.py

```diff
@@ -912,6 +912,61 @@ def get_osd_utilization():
     return osd_filled
 
 
+def get_ceph_df_detail():
+    """
```

Comment:
please change to sphynx docstring style - https://www.jetbrains.com/help/pycharm/using-docstrings-to-specify-types.html#example

Reply:
Fixed

Reply:
still not fixed

Reply:
@shyRozen See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/ how the expected doc string format looks like.

Reply:
I think now it should be ok. Thanks @mbukatov for the clarification.